### ISSUE TITLE
Add simple option to create possible retrofit combinations for given archetyp

### ISSUE THREAD

**Issue by FWuellhorst** [RWTH-EBC/TEASER#774]
To study the impact of single retrofit measures (e.g. rooftop etc.) on the building envelope and energy system, studying possible combinations is often done.
This feature adds a simple option to generate buildings to project.py.

**Comment by PGorzalka** [comment]
@FWuellhorst have you already worked on this? My work on #679 includes the possibility to load specific elements by code, that might come handy for this task. You might want to have a look at [https://github.com/PGorzalka/TEASER/tree/issue679_custom_paths_interzonal_soil](the code in my fork).
I'm currently doing the final tests before opening the pull request here. The OpenModelica compatibility regarding AixLib issue 1080 that I contacted you about a few weeks ago is already solved.

**Comment by FWuellhorst** [comment]
@PGorzalka : Yes, its already finished, the PR is currently under review: https://github.com/RWTH-EBC/TEASER/pull/775
Nice to hear :)

**Comment by PGorzalka** [comment]
Thanks, that's good to see. I'll have a look and align my contributions with the way it's coded there.
